packages/search-token-server/.eslintrc

```diff
@@ -1,6 +0,0 @@
-{
-  "extends": "../../.eslintrc",
```

Comment:
this line breaks point to an invalid path once added to a ui project (react, angular, vue)